examples/pom.xml

```diff
@@ -54,6 +54,7 @@
     <modules>
         <module>authentication</module>
         <module>ejb-in-war</module>
+        <module>ejb-in-ear</module>
```

Comment:
Perhaps add it before ejb-in-war to keep alphebetical order. I realize that jkube should be moved too.

Reply:
Fixed, all of them are sorter